pkg/audit/manager.go

```diff
@@ -406,7 +415,7 @@ func (am *Manager) auditResources(
 						log.Error(err, "error while marshaling unstructured object to JSON")
 						continue
 					}
-					if err := os.WriteFile(destFile, []byte(data), 0o600); err != nil {
+					if err := os.WriteFile(destFile, data, 0o600); err != nil {
```

Comment:
> cannot use data (variable of type string) as type []byte in argument to os.WriteFile

Reply:
Good find

Reply:
data is of type []byte, as item.MarshalJSON returns []byte.